### PR TITLE
Added code coverage report and badge

### PR DESCRIPTION
**Pull request by DonCallisto**
See https://github.com/PUGX/badge-poser/issues/394

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ or its latest stable version.
 [![License](https://poser.pugx.org/pugx/badge-poser/license.svg)](https://packagist.org/packages/pugx/badge-poser)
 [![Downloads](https://poser.pugx.org/pugx/badge-poser/d/total.svg)](https://packagist.org/packages/pugx/badge-poser)
 [![composer.lock available](https://poser.pugx.org/pugx/badge-poser/composerlock)](https://packagist.org/packages/pugx/badge-poser)
+[![Coverage Status](https://coveralls.io/repos/github/PUGX/badge-poser/badge.svg?branch=code_coverage_report)](https://coveralls.io/github/PUGX/badge-poser?branch=code_coverage_report)
```
Comment (DonCallisto) [inline README.md:12]: the `branch=` query parameter (copy/pasted this from coveralls directly) should maybe pointed to `master`?
WDYT?

Comment (AlessandroMinoccheri) [reply README.md:12]: For me `master` is the correct branch.

Comment (DonCallisto) [reply README.md]: Updated to `mater`.

**Comment by DonCallisto** [comment]
One thing that's worth to notice: Coveralls requires a PHP dependency in order to upload coverage file (https://github.com/PUGX/badge-poser/pull/397/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R48)

AFAIK, codecov is better integrated (actually not sure; didn't used in a while) but I didn't had the chance to add this repo in order to generate a secret for badge-poser. If someone of the owners can add this in codecov, we surely need to try it and see if is better.
My only concern is that coveralls seems to be less used than codecov and to have a dependency written in PHP could "block" development process (ie.: composer conflicts) if the library won't be updated regularly.

**Comment by AlessandroMinoccheri** [comment]
Be can try to use codecov. What do you think @JellyBellyDev ?

**Comment by JellyBellyDev** [comment]
> AFAIK, codecov is better integrated (actually not sure; didn't used in a while) but I didn't had the chance to add this repo in order to generate a secret for badge-poser. If someone of the owners can add this in codecov, we surely need to try it and see if is better.

I am fully available to generate the key and insert it into circleci envs!

> Be can try to use codecov. What do you think @JellyBellyDev ?

as mentioned [here](https://github.com/PUGX/badge-poser/issues/392#issuecomment-706948535) I have no preferences about it!

**Comment by DonCallisto** [comment]
@JellyBellyDev ok, once generated and inserted, let me know (via Slack?) so I can proceed and try with codecov.
Thanks.

**Comment by JellyBellyDev** [comment]
> @JellyBellyDev ok, once generated and inserted, let me know (via Slack?) so I can proceed and try with codecov.
> Thanks.

I've generate token and put it into envs for the circleCI. You can use in the pipeline this env name `$CODECOV_TOKEN`;)

**Comment by DonCallisto** [comment]
Thanks!

**Comment by DonCallisto** [comment]
Updated this PR in order to use codecov.
It seems to work.
Badge will be updated when code is merged into master branch.

**Review by JellyBellyDev — APPROVED** [review]
AWESOME!!!!

LGTM

**Pull request merged**